src/test/java/io/stargate/sgv2/jsonapi/api/v1/InsertIntegrationTest.java

```diff
@@ -751,13 +750,10 @@ public void orderedDuplicateDocumentNoNamespace() {
           .post(CollectionResource.BASE_PATH, "something_else", collectionName)
           .then()
           .statusCode(200)
-          .body("status.insertedIds", is(empty()))
+          .body("status.insertedIds", is(nullValue()))
           .body("data", is(nullValue()))
-          .body(
-              "errors[0].message",
-              startsWith(
-                  "Failed to insert document with _id 'doc4': INVALID_ARGUMENT: keyspace something_else does not exist"))
-          .body("errors[0].exceptionClass", is("StatusRuntimeException"));
+          .body("errors[0].message", startsWith("The provided namespace does not exist"))
```

Comment:
Minor improvement request: check that namespace name is matched too (since it is now included).